pallas-primitives/src/alonzo/model.rs

```diff
@@ -142,7 +144,7 @@ pub enum InstantaneousRewardSource {
 
 #[derive(Serialize, Deserialize, Debug, PartialEq, Eq, PartialOrd, Ord, Clone)]
 pub enum InstantaneousRewardTarget {
-    StakeCredentials(KeyValuePairs<StakeCredential, i64>),
+    StakeCredentials(BTreeMap<StakeCredential, i64>),
     OtherAccountingPot(Coin),
```

Comment:
It seems that this `KeyValuePairs` (at least) was really a proxy for literally an array of pairs (that encodes as such), and not really a map.

Is this correct? Should I used such an array here instead of `BTreeMap`?